.vscode/settings.json

```diff
@@ -1,6 +1,10 @@
 {
-	"python.linting.mypyEnabled": true,
-	"python.linting.enabled": true,
-	"python.testing.unittestEnabled": false,
-	"python.testing.pytestEnabled": true
+  "python.linting.mypyEnabled": true,
+  "python.linting.enabled": true,
+  "python.testing.unittestEnabled": false,
+  "python.testing.pytestEnabled": true,
+  "[python]": {
+    "editor.defaultFormatter": "ms-python.black-formatter"
+  },
+  "python.formatting.provider": "black"
```

Comment:
Does anyone take issue w/ this? It's what VSCode suggested and i think it makes sense